server/src/store.ts

```diff
@@ -1,16 +1,57 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
+import axios from 'axios';
+// import VueResource from 'vue-resource';
 
-Vue.use(Vuex);
+Vue.use( Vuex );
 
 export default new Vuex.Store({
-  state: {
+  // Same as 'data' for a Vue component, except accessable by all components.
+  state: { // {{{
+    login: {
+      username: '',
+      password: '',
+      response: {},
```

Comment:
Does all state that we want to save need to go in here?

Reply:
Yes, if you want it to persist. But we can break the store into multiple files.